src/main/java/core/basesyntax/strategy/SupplyHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class SupplyHandler implements OperationHandler {
+    private static final String SUPPLY = "s";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
```

Comment:
The method signature `getFruitFinalQuantity` suggests a more general usage, but the implementation is specific to the 'supply' operation. Consider renaming the method to reflect its specific purpose or refactoring the method to be more generic.